app/models/notifications/queries/MucrInfo.scala

```diff
@@ -25,7 +25,11 @@ case class MucrInfo(
   entryStatus: Option[EntryStatus] = None,
   isShut: Option[Boolean] = None,
   movements: Seq[MovementInfo] = Seq.empty
-)
+) {
+  def transport: Option[Transport] =
+    movements.find(movementInfo => movementInfo.transportDetails.isDefined).flatMap(movementWithTransport => movementWithTransport.transportDetails)
```

Comment:
Probably you can replace `movementInfo` and `movementWithTransport` with `_`

Reply:
I find it easier to deal with mappings if I create a variable while I'm typing.  I usually then replace with "_" when possible.   I'll fix this.